docs/markdown/Using Pants/concepts/enabling-backends.md

```diff
@@ -16,43 +16,100 @@ backend_packages = [
+| `pants.backend.python.lint.pyupgrade`                | Enables Pyupgrade, which upgrades to new Python syntax: <https://pypi.org/project/pyupgrade/>      | [Linters and formatters](doc:python-linters-and-formatters)       |
+| `pants.backend.python.lint.yapf`                     | Enables Yapf, the Python formatter: <https://pypi.org/project/yapf/>                               | [Linters and formatters](doc:python-linters-and-formatters)       |
+| `pants.backend.python.typecheck.mypy`                | Enables MyPy, the Python type checker: <https://mypy.readthedocs.io/en/stable/>.                   | [typecheck](doc:python-check-goal)                                |
+| `pants.backend.shell`                                | Core Shell support, including shUnit2 test runner.                                                 | [Shell overview](doc:shell)                                       |
+| `pants.backend.shell.lint.shfmt`                     | Enables shfmt, a Shell autoformatter: <https://github.com/mvdan/sh>.                               | [Shell overview](doc:shell)                                       |
+| `pants.backend.shell.lint.shellcheck`                | Enables Shellcheck, a Shell linter: <https://www.shellcheck.net/>.                                 | [Shell overview](doc:shell)                                       |
+| `pants.backend.tools.preamble`                       | Enables "preamble", a Pants fixer for copyright headers and shebang lines                          | [`preamble`](doc:reference-preamble)                              |
+| `pants.backend.tools.taplo`                          | Enables Taplo, a TOML autoformatter: <https://taplo.tamasfe.dev>                                   |                                                                   |
+| `pants.backend.url_handlers.s3`                      | Enables accessing s3 via credentials in `file(source=http_source(...))`                            |                                                              |
+
+Available experimental backends
+-------------------------------
+
+Pants offers [additional backends as previews](https://blog.pantsbuild.org/quick-feedback-on-new-features-via-experimental-backends/) that are still in development. These backends may still undergo major changes to improve the interface or fix bugs, with fewer (or no) deprecation warnings. If any of these backends are relevant to you, please try them, [ask any questions](doc:getting-help) you have, and [contribute improvements](doc:contributor-overview)! Volunteers like you jumping in to help is how these backends are promoted from preview to fully stable.
+
+This list is also available via `pants backends --help-advanced`, which includes any additional plugins in your repository that aren't built-in to Pants itself.
```

Comment:
nit: the advanced help includes all backends, not only the experimental ones, so this phrasing works perfect for the first table, but feels slightly off for the second table.

Reply:
```suggestion
The list of all backends (both stable and experimental) is also available via `pants backends --help-advanced`, which includes any additional plugins in your repository that aren't built-in to Pants itself.
```